cmd/ctr/commands/run/run.go

```diff
@@ -170,6 +170,10 @@ var Command = cli.Command{
 			Name:  "detach,d",
 			Usage: "detach from the task after it has started execution",
 		},
+		cli.StringFlag{
+			Name:  "fifo-dir",
```

Comment:
Do we use "Dir" anywhere else? Should this be path?

Reply:
We have `ctr run --cwd` and `containerd --workdir`, so I think `dir` is fine.